ckanext-notify/README.rst

```diff
@@ -0,0 +1,166 @@
+
+.. image:: https://pypip.in/download/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi//ckanext-notify/
+    :alt: Downloads
+
+.. image:: https://pypip.in/version/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: Latest Version
+
+.. image:: https://pypip.in/py_versions/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: Supported Python versions
+
+.. image:: https://pypip.in/status/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: Development Status
+
+.. image:: https://pypip.in/license/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: License
+
+=============
+ckanext-notify
+=============
+
+.. Put a description of your extension here:
+   What does it do? What features does it have?
+   Consider including some screenshots or embedding a video!
+
+
+------------
+Requirements
+------------
+
+For example, you might want to mention here which versions of CKAN this
+extension works with.
+
+
+------------
+Installation
+------------
+
+.. Add any additional install steps to the list below.
+   For example installing any non-Python dependencies or adding any required
+   config settings.
+
+To install ckanext-notify:
+
+1. Activate your CKAN virtual environment, for example::
+
+     . /usr/lib/ckan/default/bin/activate
+
+2. Install the ckanext-notify Python package into your virtual environment::
+
+     pip install ckanext-notify
+
+3. Add ``notify`` to the ``ckan.plugins`` setting in your CKAN
+   config file (by default the config file is located at
+   ``/etc/ckan/default/production.ini``).
+
+4. Restart CKAN. For example if you've deployed CKAN with Apache on Ubuntu::
+
+     sudo service apache2 reload
+
+
+---------------
+Config Settings
+---------------
+
+Document any optional config settings here. For example::
+
+    # The minimum number of hours to wait before re-checking a resource
+    # (optional, default: 24).
+    ckanext.notify.some_setting = some_default_value
+
+
+------------------------
+Development Installation
+------------------------
+
+To install ckanext-notify for development, activate your CKAN virtualenv and
+do::
+
+    git clone https://github.com/"WNjihia"/ckanext-notify.git
+    cd ckanext-notify
+    python setup.py develop
+    pip install -r dev-requirements.txt
+
+
+-----------------
+Running the Tests
+-----------------
+
+To run the tests, do::
+
+    nosetests --nologcapture --with-pylons=test.ini
+
+To run the tests and produce a coverage report, first make sure you have
+coverage installed in your virtualenv (``pip install coverage``) then run::
+
+    nosetests --nologcapture --with-pylons=test.ini --with-coverage --cover-package=ckanext.notify --cover-inclusive --cover-erase --cover-tests
+
+
+---------------------------------
+Registering ckanext-notify on PyPI
```

Comment:
Should these be part of the Readme, Does this imply your expectation is to expect more people to add ckanext-notify on pypi, if we have developed it and added to pypi and is installable with pip, should that not be enough, my thought process is that the readme should be more of how your users make use of ckanext-notify and not how you added it to pypi.